packages/@aws-cdk/aws-kinesis/README.md

```diff
@@ -192,3 +193,74 @@ const stream = new Stream(stack, 'MyStream');
 // give my user permissions to list shards
 stream.grant(user, 'kinesis:ListShards');
 ```
+
+### Metrics
+
+You can use common metrics from your stream to create alarms and/or dashboards.
+Read more about Kinesis metrics at [Creating and Managing Streams](https://docs.aws.amazon.com/streams/latest/dev/working-with-streams.html)
+
+You can find bellow a example of building a dashboard with the default metrics.
```

Comment:
No need, it's a little overwhelming to look at. We don't need to explain how to create dashboards, we just need to mention that metrics are available using `stream.metricXXX` methods and show an example or two.